src/pages/Transfer/Transfer.jsx

```diff
@@ -1,18 +1,65 @@
 import "./Transfer.scss";
+import Button from "../../components/Button/Button";
+const Transfer = ({
```

Comment:
add new line between imports and declaring component

Reply:
done